### ISSUE TITLE
what happens before epoch 10

### ISSUE THREAD

**Issue by flobotics** [ashawkey/stable-dreamfusion#150]
### Description

When i run the same text-prompt with different settings , the train/loss value before epoch 10 has nearly no big ups/downs , then they start to go up and down.

<img width="1155" alt="tensorboard2" src="https://user-images.githubusercontent.com/11964445/219523118-1afe264a-9ca9-40d9-94ac-4d60be94f449.png">

### Steps to Reproduce

run the same text-prompt with different settings

### Expected Behavior

the train/loss value should directly at start go more up and down, like they do after epoch 10

### Environment

win 11

**Comment by ashawkey** [comment]
Hi, SDS doesn't have an explicit loss value, so the training loss here is just the regularization loss (majorly orientation loss), which only starts to change after 10 epochs (i.e., after the albedo-only training stage, you could change this through `--albedo_iters`).
